Jenkinsfile

```diff
@@ -6,10 +6,9 @@ buildPlugin(
   useContainerAgent: true,
   // Show failures on all configurations
   failFast: false,
-  // Test Java 8 with default Jenkins version, Java 11 with a recent LTS, Java 17 even more recent
+  // Test Java 11 with default Jenkins version, Java 17 with more recent LTS
   configurations: [
-    [platform: 'linux',   jdk: '17', jenkins: '2.375'],
-    [platform: 'linux',   jdk: '11', jenkins: '2.361.4'],
+    [platform: 'linux',   jdk: '17', jenkins: '2.387.1'],
     [platform: 'windows', jdk: '11']
```

Comment:
Mark,

Is there documentation somewhere about why we should use this combination?
I already know of your "[Improve a plugin tutorial](https://www.jenkins.io/doc/developer/tutorial-improve/add-a-jenkinsfile/ )", and I know about the latest LTS, so I guess that's where it comes from.
Am I right?
Thanks.